### ISSUE TITLE
 [Bug]  clicking [[wikilinks]] keep opening new pane// [Feature request] allow to open  [[wikilink ]] in same pane similar to vanilla Obsidian

### ISSUE THREAD

**Issue by danieltomasz** [mgmeyers/obsidian-kanban#78]
I really like this plugin, but I have problem when clicking on wiki link to other note.

In Obsidian preview mode clicking on note open it in the same pane, `Ctr` + click open note in new pane.

In Kanban Plugin up to Version 0.2.3 clicking  and Ctrl + Click bring the same behaviour - opening new pane. In  a newer version Ctrl + Click will edit the card.

There is no way to open a wikilink "in place" .

Example

Pin Kanban Note with card containing wiki link.
Click several times on card containing wiki link.
Current behaviour:
(Note will open several times, every time in new pane)
![image](https://user-images.githubusercontent.com/7980381/117827799-96717a80-b271-11eb-9909-198efaa68a12.png)

Expected:
(apart pinned note, that doesn't change, only one additional pane is created, all wiki links are opened in it )
![image](https://user-images.githubusercontent.com/7980381/117827967-b86afd00-b271-11eb-834a-40fd2864dac8.png)

Solution:
Have an additional  way to open wiki link in card the same way as standard obsidian "click wiki link" work (in same pane);

Linux PopOS
Obsidian 0.12.3

Edit:
I was able to observe erratic behaviour in Obsidian "Click link"  with Kanban plugin activated, or deactivated but without restarting Obsidian.

Basically "only click"(Open a new note in  the same pane) in preview mode  sometimes acted as "Ctrl +click"(opening  a note in a new pane).  While experiencing this, after once using "Ctrl + click"  the next  opening  of notes using "Click" were normal (it opened notes in the same pane)

After restarting Obsidian with Kanban plugin deactivated I was not able to observe this behaviour (I don't use that often preview mode, so I dont know if this may be a genuine Obsidian bug)

**Comment by danieltomasz** [comment]
Obsidian has its problems but the Kanban plugin doesn't handle well clicking - ctrl + clicking on the card  with the wikilink will  edit the card instead of  opening the link in the new pane as I expected;

Activating the plugin may actually  sometimes change how Obdsiadian hand click on wikilink on normal note (this is my hypothesis, I don't remember problems with clicking links before I started using the plugin
